src/Native/build-native.cmd

```diff
@@ -66,18 +66,29 @@ if not exist "%_VSCOMNTOOLS%" goto :MissingVersion
 call "%_VSCOMNTOOLS%\VsDevCmd.bat"
 
 :RunVCVars
-if "%VisualStudioVersion%"=="15.0" (
+if "%VisualStudioVersion%"=="16.0" (
+    goto :VS2019
+)
+else if "%VisualStudioVersion%"=="15.0" (
     goto :VS2017
 ) else if "%VisualStudioVersion%"=="14.0" (
```

Comment:
Please remove all 14.0 stuff here. There is an infra issue somewhere that we don't support it anymore and need to clean it up

Reply:
https://github.com/dotnet/corefx/issues/24320

Reply:
@safern would you be willing to also remove the additional VS2015 places? https://github.com/dotnet/corefx/search?q=vs2015&unscoped_q=vs2015

Reply:
I just looked at that issue. I just assigned myself. Will merge this PR as is and will work in a separate PR to do things right and not pollute this one as it seems more work than just removing those references 😄

Reply:
Sounds good. Would really like to see all VS2015 references removed :)